healdata.org/manifests/hatchery/hatchery.json

```diff
@@ -9,6 +9,8 @@
     "account_id": "252835817747",
     "workspace_type": "healworkspaces"
   },
+  "license-user-maps-dynamodb-table": "gen3-license-user-maps-prod-heal",
```

Comment:
@george42-ctds can youy remind me if we need PE's help to create this table before we deploy, or hatchery will created automatically if it doesn't exists?

Reply:
great! thanks for the confirmation, want also help to review this PR? 😉